contracts/gov/ERC721/ERC721Multiplier.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+import "../../interfaces/gov/ERC721/IERC721Multiplier.sol";
+
+contract ERC721Multiplier is IERC721Multiplier, ERC721Enumerable, ERC721URIStorage, Ownable {
+    using Counters for Counters.Counter;
+
+    uint256 public constant MULTIPLIER_PRECISION = 10**25;
+
+    string public baseURI;
+    Counters.Counter private _tokenIds;
```

Comment:
Could you please remove the `Counters` and use the bare `uint256` variable? Plus make it `public` please

Reply:
done